README.md

```diff
@@ -6,27 +6,71 @@ easy-update-metadata-dataset
+The command is idempotent, so running it either once or multiple times gives the same result.
 
 
 ARGUMENTS
 ---------
 
-          --doUpdate   Without this argument no changes are made to the repository, the default is a test mode
-                       that logs the intended changes
-      -h, --help       Show help message
-      -v, --version    Show version of this program
-    
-     trailing arguments:
-      input-file (required)   The CSV file (RFC4180) with required changes. The first line must be
-                              'FEDORA_ID,STREAM_ID,XML_TAG,OLD_VALUE,NEW_VALUE', in that order. Additional columns
-                              and empty lines are ignored.
+           --doUpdate   Without this argument no changes are made to the repository, the default is a test mode
+                               that logs the intended changes
+       -h, --help       Show help message
+       -v, --version    Show version of this program
+         
+       trailing arguments:
+           input-file (required)                     The CSV file (RFC4180) with required changes. The first line must
+                                                     be 'FEDORA_ID,STREAM_ID,XML_TAG,OLD_VALUE,NEW_VALUE', in that
+                                                     order. Additional columns and empty lines are ignored.
+           complex_values_directory (not required)   A reference to a directory containing files with the complex
+                                                     values. These files are referenced in the input-file.
+                              
+                              
+EXAMPLES
+--------
+
+###Example: update simple value in EMD
+
+This module was developed to update simple values in the EMD, like the AccessCategory. 
+`easy-update-metadata-dataset --doUpdate changeAccessCategory.csv`
+
+```csv
+FEDORA_ID,      STREAM_ID, XML_TAG,      OLD_VALUE,    NEW_VALUE
+easy-dataset:1, EMD,       accessRights, GROUP_ACCESS, OPEN_ACCESS
+```
+###Example: check update datasetState in AMD
+
+To check a potential change of the state of a dataset, use the following
+`easy-update-metadata-dataset changeDatasetState.csv`
+
+```csv
+FEDORA_ID,      STREAM_ID, XML_TAG,      OLD_VALUE, NEW_VALUE
+easy-dataset:1, AMD,       datasetState, DRAFT,     SUBMITTED
+```
+
+when the dataset does not contain the expected value as given in `OLD_VALUE` it will produce a log line like `expected AMD <datasetState> [DRAFT] but found [PUBLISHED]`. No values will be changed, because the `--doUpdate` arguments is missing.
+
+###Example: Adding complex content
+
+`easy-update-metadata-dataset --doUpdate changeContributors.csv ~/contributorSnippets/`
+
+```csv
+FEDORA_ID,      STREAM_ID, XML_TAG,         OLD_VALUE, NEW_VALUE
+easy-dataset:1, EMD,       emd:contributor, NILL,      contributor_1.xml
+```
+
+when the `OLD_VALUE` is`NILL`, the value in `NEW_VALUE` must be __added__ to the tag 
```

Comment:
Can this (new) `NIL` feature also be used apart from the `complex-values-directory`? If so, please make it a separate example.